external-crates/move/crates/move-analyzer/trace-debug/package.json

```diff
@@ -1,156 +1,179 @@
 {
-  "name": "move-trace-debug",
-  "displayName": "Move Trace Debugger",
-  "description": "An extension to visualize Move VM traces DAP-style",
-  "publisher": "mysten",
-  "icon": "images/move.png",
-  "license": "Apache-2.0",
-  "version": "0.0.5",
-  "preview": true,
-  "repository": {
-    "url": "https://github.com/MystenLabs/sui.git",
-    "type": "git"
-  },
-  "engines": {
-    "vscode": "^1.92.0"
-  },
-  "categories": [
-    "Other"
-  ],
-  "activationEvents": [
-    "onDebugDynamicConfigurations:move-debug",
-    "onDebugResolve:move-debug"
-  ],
-  "main": "./out/extension.js",
-  "contributes": {
-    "commands": [
-        {
-          "command": "move.toggleDisassembly",
-          "title": "Toggle disassembly view",
-          "category": "Move"
-        },
-        {
-          "command": "move.toggleSource",
-          "title": "Toggle source view",
-          "category": "Move"
-        }
+    "name": "move-trace-debug",
+    "displayName": "Move Trace Debugger",
+    "description": "An extension to visualize Move VM traces DAP-style",
+    "publisher": "mysten",
+    "icon": "images/move.png",
+    "license": "Apache-2.0",
+    "version": "0.0.5",
+    "preview": true,
+    "repository": {
+        "url": "https://github.com/MystenLabs/sui.git",
+        "type": "git"
+    },
+    "engines": {
+        "vscode": "^1.92.0"
+    },
+    "categories": [
+        "Other"
     ],
-    "languages": [
-        {
-          "id": "move",
-          "aliases": [
-            "Move",
-            "move"
-          ],
-          "extensions": [
-            ".move"
-          ]
-        },
-        {
-          "id": "mvb",
-          "aliases": [
-            "MVB",
-            "mvb"
-          ],
-          "extensions": [
-            ".mvb"
-          ]
-        },
-        {
-          "id": "mtrace",
-          "aliases": [
-            "mtrace"
-          ],
-          "extensions": [
-            ".json",
-            ".JSON"
-          ]
-        }
-      ],
-    "breakpoints": [{ "language": "move" }, { "language": "mvb" }],
-    "debuggers": [
-      {
-        "type": "move-debug",
-        "label": "Move Debug",
-        "program": "./out/server.js",
-        "runtime": "node",
-        "runtimeArgs": [
-          "--nolazy"
+    "activationEvents": [
+        "*"
```

Comment:
This is an actual change. Having a star here means that the extension activates immediately after VSCode starts which is normally not ideal but:
- we need it here to properly handle custom display of compressed trace files (we need to register custom handlers in `activate` function but there is no other explicit trigger that would caused this function to be called when opening a binary file)
- not much happens on activation of this extension so it should have minimal impact on the user experience